pypeit/find_objects.py

```diff
@@ -986,7 +986,7 @@ def initialize_slits(self, slits, initial=True):
                 Use the initial definition of the slits. If False,
                 tweaked slits are used.
         """
-        super().initialize_slits(slits, initial=True)
+        super().initialize_slits(slits)
```

Comment:
If initial is always going to be false, I would add this to the docstring.  I.e., the user needs to know that if they set `initial=True` the code is going to ignore it.  But, if you think there may be a reason to set `initial=True`, you might consider removing this method so that the class always uses the base class method exactly.